src/content/en/tools/workbox/reference-docs/latest/_toc.yaml

```diff
@@ -1,57 +1,57 @@
 toc:
 - title: "BaseCacheManager"
-  path: /web/tools/workbox/reference-docs/latest/module-workbox-precaching.BaseCacheManager
+  path: /web/tools/workbox/reference-docs/v2.0.3/module-workbox-precaching.BaseCacheManager
```

Comment:
Pardon a naive question: with each generation of new reference docs, will we be preserving docs for previous releases (WF repo including each version) or removing them? (e.g 2.0.3, 2.0.4 etc)

Reply:
At the moment there is nothing that will remove them but the build process is configured to old look for and build the latest tagged release.